static/src/javascripts/projects/commercial/modules/prebid/bid-config.js

```diff
@@ -217,7 +217,38 @@ const getImprovePlacementId = (sizes: PrebidSize[]): number => {
     }
 };
 
-const getAppNexusDirectPlacementId = (): string => '11016434';
+const getAppNexusDirectPlacementId = (sizes: PrebidSize[]): string => {
+    if (isInAuRegion()) {
+        return '11016434';
+    }
+
+    const defaultPlacementId: string = '9251752';
+    switch (getBreakpointKey()) {
+        case 'D':
+            if (containsMpuOrDmpu(sizes)) {
+                return '9251752';
```

Comment:
`return defaultPlacementId;` ???

Reply:
Yep, Desktop MPU is the default placement ID, which should never happen. Could use the variable but didn't as wanted it to be clear where the default is coming from.